api/envoy/extensions/transport_sockets/quic/v3/quic_transport.proto

```diff
@@ -20,6 +22,10 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 message QuicDownstreamTransport {
   tls.v3.DownstreamTlsContext downstream_tls_context = 1
       [(validate.rules).message = {required: true}];
+
+  // If false, QUIC will tell TLS to reject any early data and to stop issuing 0-RTT credentials with resumption session tickets. So that the clients won't attempt 0-RTT requests.
+  // Default to true.
+  google.protobuf.BoolValue enable_early_data = 2;
```

Comment:
I would be fine with this being `bool disable_early_data` as well, but up to you.

Reply:
Sounds good to me. If earlier data is enabled by default, then having to explicitly set `disable_early_data = true` seems to express the right semantics.

Reply:
Also, please add version history update for this knob. Thanks.

Reply:
Using "enable" rather than "disable" here is to be consistent with QuicSSLConfig::early_data_enabled.